src/buildercore/trop.py

```diff
@@ -148,7 +148,14 @@ def mkoutput(title, desc, val):
 
 def build_vars(context, node):
     buildvars = dict(context)
-    del buildvars['project']
+
+    # preseve some of the project data. all of it is too much
```

Comment:
yeah, I think we've talked about the buildvars before and keeping them minimal. I ended up going with passing the formulas to the init-formulas.sh script, but originally I was going to pull them from the buildvars. I'm still  undecided whether I should store the pinned revisions for the formulas in the buildvars  or even store them at all.